search_bar.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css">
+
+</head>
+<body>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+        <label>
+            <div class="input">
+                    <input 
+                      type="search" 
+                      data-qa="keypress" 
+                      data-qa="big" 
+                      class="placeholder" 
+                      placeholder="Try ''Los Angeles''">      
+              </div>
+            </label>
+
+            </form>
+
+            <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+                <label>
+                <div class="input inputSmall">   
+                        <input 
+                          type="search" 
+                          data-qa="small" 
+                          class="placeholder_1" 
+                          placeholder="Try ''Los Angeles''">
+                </div>
+            </label>
+
+            </form>  
+  </body>
+</html>
```

Comment:
```suggestion
</html>

```
Тут потрібно додати пустий рядок